tools/make-srpm

```diff
@@ -51,12 +51,7 @@ rpmbuild "${rpmbuild_args[@]}" -ts ${source} | sed -n 's@^Wrote:.*/@@p'
 # "cockpit" with build_basic=1, and "cockpit-appstream" with build_optional=1
 # the spec has proper build_* defaults depending on the Name:
 if grep -q 'platform:el8' /usr/lib/os-release; then
-    appstream_spec="${tmpdir}/cockpit-appstream.spec"
-    tar xf "$source" -O '*/tools/cockpit.spec' |
-        sed '/^Name:/ s/$/-appstream/' > "${appstream_spec}"
```

Comment:
FTR: I did it that way because it's pretty much exactly the process which we do downstream. I'm a bit nervous about this breaking our release scripts and places which sed/grep/scan the spec, but no veto if it works.